packages/components/src/components/select-wrapper/select-wrapper-dropdown/select-wrapper-dropdown.tsx

```diff
@@ -182,56 +207,68 @@ export class SelectWrapperDropdown {
               {this.description}
             </div>
           ),
-          this.isOpen && (
-            <ul
-              id={dropdownId}
-              role="listbox"
-              tabIndex={-1}
-              {...getListAriaAttributes(this.label, this.required, this.filter, this.isOpen)}
-              ref={(el) => (this.listElement = el)}
-            >
-              {this.filter && !hasFilterResults(this.optionMaps) ? (
-                <li class="option" aria-live="polite" role="status">
-                  <span aria-hidden="true">---</span>
-                  <span class="option__sr">No results found</span>
-                </li>
-              ) : (
-                this.optionMaps.map((option, index) => {
-                  const { value, disabled, hidden, initiallyHidden, selected, highlighted, title } = option;
-                  return [
-                    title && (
-                      <span class="optgroup" role="presentation">
-                        {title}
-                      </span>
-                    ),
-                    <li
-                      id={`option-${index}`}
-                      role="option"
-                      class={{
-                        option: true,
-                        'option--selected': selected,
-                        'option--highlighted': highlighted,
-                        'option--disabled': disabled,
-                        'option--hidden': hidden || initiallyHidden,
-                      }}
-                      onClick={!selected && !disabled ? () => this.setOptionSelected(index) : undefined}
-                      {...getOptionAriaAttributes(selected, disabled, hidden, !!value)}
-                    >
-                      {value}
-                      {selected && !disabled && (
-                        <PrefixedTagNames.pIcon
-                          aria-hidden="true"
-                          name="check"
-                          color={disabled ? 'state-disabled' : 'primary'}
-                          theme={this.theme}
-                        />
-                      )}
-                    </li>,
-                  ];
-                })
-              )}
-            </ul>
-          ),
+          <div
+            {...(this.isNativePopover && {
+              popover: 'auto',
+              id: 'popover',
+              class: 'popover',
+              ...(this.popoverElement?.matches(':popover-open') && {
+                'popover-open': true,
+              }),
+            })}
+            ref={(el) => (this.popoverElement = el)}
+          >
+            {this.isOpen && (
+              <ul
```

Comment:
Can't this be made a native popover when needed? 
Or do we always need the wrapping div?

Reply:
I tried this first, but unfortunately the `ul` element doesn't exist until the list is in open state. And the browser can't do its portal "magic" anymore. So we need the wrapping div with `popover` attribute.